samples/cpp/benchmark_app/main.cpp

```diff
@@ -938,7 +944,17 @@ int main(int argc, char* argv[]) {
                 // method of `std::exception` So, rechecking for any exceptions here.
                 inferRequest->wait();
                 inferRequest->start_async();
-            }
+                // setting target duration allow to get required number of FPS.
+                // if number of required FPS is not set it will be calculated from reported duration and number of
+                // iterations.
+                if (fps_number > 0) {
+                    double targetDuration = double(batchSize * 1000 / fps_number);
+                    auto exec = std::chrono::duration_cast<ns>(Time::now() - startTime2).count() * 0.000001;
+                    double difference = targetDuration - exec;
+                    if (difference > 0) {
+                        Sleep(unsigned long(difference));
```

Comment:
is it Windows-specific code?

Reply:
No, it should be generic, independent from platform.

Reply:
Ok, in case of failures, please consider using std::this_thread::sleep_for(<time>)